NetworkConfigurations/NetworkAddressingAndFilteringSubCategories/Stateless-NAT.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Stateless Network Address Translation
+grand_parent: Network Configuration
+parent: Network Addressing and Filtering
+nav_order: 4
+layout: default
+---
+
+# Stateless Network Address Translation (NAT)
+
+Stateless Network Address Translation (NAT), also known as static NAT, establishes a permanent mapping between a private IP address and a single public address.
+
+To configure stateless NAT, the `tc iproute2` tool is utilized, along with the `tc-nat` action. This action is combined with the `flower` filter rule and `ingress` qdisc to enable static NAT entry offloading. Below is the command format:
```

Comment:
I do not think these are right. Or at least I do not see these actions used in the doc sections below. Could you please double check?

Reply:
Nevermind, I do see action nat. Please add a graph of sample topology (you can use VMs) and verify the config works before deleting/modifying anything.